docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -203,7 +203,7 @@ Trino data type mapping:
     - ``TIMESTAMP(0)``
     - See :ref:`datetime mapping`
   * - ``TIMESTAMP(p)``
-    - ``TIMESTAMP``
+    - ``TIMESTAMP(p)``
```

Comment:
> Adding support for Oracle timestamp with various precision

Please update the commit title as "Add support for Oracle timestamp with various precision"
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages
> Use the imperative mood in the subject line

Reply:
refactored